### ISSUE TITLE
[Question about scrolling]autoscrolling cannot have different speed

### ISSUE THREAD

**Issue by franklingu** [nicklockwood/iCarousel#627]
Not sure if I understand this correct but it seems that scroll speed does not related to autoscrolling

another question: is it possible to have rotated linear? meaning that from view 1 to view 2 to view 3 to view 1?

**Comment by nicklockwood** [comment]
That's correct. Autoscrolling speed is controlled purely by the value of the autoscroll property.

To enable wrapping for a linear carousel, use the carousel:valueForOption: delegate method.

**Comment by franklingu** [comment]
thanks. cause a lot of CGFloat value confused me. the value for autoscroll is in fact the speed right?

**Comment by nicklockwood** [comment]
Correct.

**Comment by HadiYaac** [comment]
Changing the scrollSpeed property is not affecting the scroll speed!
